core/src/main/java/hudson/PluginManager.java

```diff
@@ -1182,22 +1198,30 @@ private void trackInitialPluginInstall(@Nonnull final List<Future<UpdateCenter.U
             new Thread() {
                 @Override
                 public void run() {
+                    final boolean failures[] = { false };
```

Comment:
Yes, also applicable to hypothetical parameters of `run`, but not local variables.